BigCommerce/UserInterface/01a_Connections.md

```diff
@@ -0,0 +1,198 @@
+
+- *Access token*   
+    Access token for the *BigCommerce* connection. This field is automatically filled after having created the *BigCommerce* connection automatically.
+
+- [SAVE]  
+    Click this button to save the connection. 
+
+
+
+## Edit BigCommerce connection
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Select BigCommerce connection*
+
+![Edit Shopify connection](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/EditConnectionCredentials.png "[Edit Shopify connection]")
+
+- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)   
+    Click this button to close the *Edit connection* view and return to the connection list. All changes are rejected.
+
+- *Name*   
+    Connection name. 
+
+- ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit)  
+    Click this button to edit the connection name.
+
+- ![Apply](../../Assets/Icons/Check.png "[Apply]") (Apply)  
+    Click this button to apply the changes to the connection name.  This button is only displayed if you are editing the connection name.
+
+- *Created DD/MM/YYYY*  
+    Creation date of the connection. This field is read-only.  
+
+
+
+## Edit BigCommerce connection &ndash; Credentials
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Select BigCommerce connection > Tab Credentials*
+
+![Edit BigCommerce credentials](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/EditConnectionCredentials.png "[Edit BigCommerce credentials]")
+
+- *Driver*  
+    Driver name. For *BigCommerce*, the *BigCommerce* driver is displayed. This drop-down list is read-only.
+
+**Credentials** 
+
+- *Store hash*   
+    Click the field to edit the hash key of the *BigCommerce* store.
+
+- *Access token*   
+    Click the field to edit the access token for the *BigCommerce* connection.
+
+- [SAVE]  
+    Click this button to save any changes made.
+
+
+
+## Edit BigCommerce connection &ndash; Settings
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Select BigCommerce connection > Tab Settings*
+
+![Edit BigCommerce settings](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/EditConnectionSettings.png "[Edit BigCommerce settings]")
+
+The *Settings* tab is composed of the following setting entries:
+- [Variant option type](#variant-option-type)
+- [Order filters](#order-filters)
+
+
+### Variant option type
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Select BigCommerce connection > Tab Settings > Menu entry Variant option type*
+
+![Variant option type](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/VariantOptionType.png "[Variant option type]")
+
+- *Variant option type used for variants*   
+   Use this dropdown list to select the type to select variants. For detailed information, see [Configure BigCommerce connection](../Integration/01_ManageBigCommerceConnection.md#configure-bigcommerce-connection).
```

Comment:
```suggestion
   Use this drop-down list to select the type to select variants. For detailed information, see [Configure BigCommerce connection](../Integration/01_ManageBigCommerceConnection.md#configure-bigcommerce-connection).
```
"to select the type to select variants"... Evtl. umformulieren, um Wiederholung zu vermeiden.

Reply:
done